### PR TITLE
feature: Add support for pii types and column scanning to deep scan 

### PR DESCRIPTION
**Pull request by nicolepng**

### CODE REVIEW

**Comment by nicolepng** [comment]
related to issue#68

File: piicatcher/scanner.py [hunk 1]
```diff
@@ -161,6 +171,9 @@ def data_scan(
         LOGGER.debug("Scanning column name %s", column.fqdn)
         if val is not None:
             for detector in detectors:
+                if column.pii_type is not None:
+                    continue
```
Comment (jhecking) [inline piicatcher/scanner.py:175]: Does this mean if the metadata scan has already determined the PII type of the column, then the data of that column will not be scanned any more?

Comment (nicolepng) [reply piicatcher/scanner.py:175]: yup, but not sure if that should be the default or allow the datum scanning to override the piitype

File: piicatcher/api.py [hunk 2]
```diff
@@ -90,43 +90,45 @@ def scan_database(
...
             )
 
-            if scan_type == ScanTypeEnum.metadata:
-                Stats().record_event("/pip/piicatcher", "scan_type: {}".format(scan_type))
-                detector_list = [
-                    detector()
-                    for detector in detectors.detector_registry.get_all().values()
-                    if issubclass(detector, MetadataDetector)
-                ]
+            # if scan_type == ScanTypeEnum.metadata:
```
Comment (jhecking) [inline piicatcher/api.py:93]: Just remove this code if it's no longer needed rather than just comment it out.

**Review by jhecking — APPROVED** [review]

**Pull request merged**